frontend/src/pages/item/_itemId/index.vue

```diff
@@ -246,13 +247,24 @@ import {
 import { getUserLibraryApi } from '@jellyfin/sdk/lib/utils/api/user-library-api';
 import { getBlurhash } from '@/utils/images';
 import { getItemDetailsLink, getMediaStreams } from '@/utils/items';
-import { getItemizedSelect } from '@/utils/forms';
+import { getItemizedSelect, VSelectItem } from '@/utils/forms';
 import { useRemote } from '@/composables';
 
 const route = useRoute();
 const remote = useRemote();
 
-const item = ref<BaseItemDto>({});
+const { itemId } = route.params as { itemId: string };
+
+const item = ref(
+  (
+    await remote.sdk.newUserApi(getUserLibraryApi).getItem({
+      userId: remote.auth.currentUserId ?? '',
+      itemId
+    })
+  ).data
+);
```

Comment:
Do we need this to be a ref?

Reply:
Indeed, doesn't seem so. I removed the ref.